tutorials/notebooks/celestial-coords1/celestial_coords1.ipynb

```diff
@@ -0,0 +1,582 @@
+    "We can print the FITS image header to screen so that all of its contents can be checked or utilized. Note that the WCS information for this information can be found near the bottom of the printed header, below."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/",
+     "height": 1000
+    },
+    "colab_type": "code",
+    "id": "XcerK_cZJ30W",
+    "outputId": "f609c0ae-0116-45c6-fd89-921a208c9434"
+   },
+   "outputs": [],
+   "source": [
+    "print(repr(header))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "g3FyYZ-bQ5b7"
+   },
+   "source": [
+    "Please note that the original header violates the FITS WCS standards (including both CDELTn and CD-matrix; including deprecated PC-matrix keywords). The header has been cleaned up to conform to the existing standards."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "FzAtb_LWLyWn"
+   },
+   "source": [
+    "### Step 2: Read in the FITS image coordinate system with astropy.wcs.WCS\n",
+    "\n",
+    "Because the header contains WCS information, the Astropy `WCS` class can be loaded directly from the FITS header information."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "3Wl5J0HBLoIp"
+   },
+   "outputs": [],
+   "source": [
+    "wcs_helix = WCS(header)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "Akv472vTZFLp"
+   },
+   "source": [
+    "Let's print the `WCS` object to see what values were drawn from the header."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/",
+     "height": 194
+    },
+    "colab_type": "code",
+    "id": "1C7y709IZNvl",
+    "outputId": "6aee1d99-9f6f-4990-ef55-aa601c33ff0d"
+   },
+   "outputs": [],
+   "source": [
+    "print(wcs_helix)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "O10H9rmtQEnM"
+   },
+   "source": [
+    "### Step 3: Plot the Helix nebula with the proper RA and Dec\n",
+    "\n",
+    "Next, let's view the image. We will be using the `matplotlib` library for the plotting. \n",
+    "\n",
+    "The `WCS` object is built so it can be provided to `matplotlib` with the  `projection` keyword, as shown in the call to `matplotlib.pyplot.subplot`, below."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/",
+     "height": 607
+    },
+    "colab_type": "code",
+    "id": "fnv48KDbOkxB",
+    "outputId": "05775d9a-6663-4b01-a3d3-0d7c9440ea64"
+   },
+   "outputs": [],
+   "source": [
+    "import matplotlib.pyplot as plt\n",
```

Comment:
Please move this to the top of the page with the rest of the imports

Reply:
Sure, done